Automattic/jetpack
# Social Menu & Social Media Icons: Add Mastodon Icon

Fixes #10338

#### Changes proposed in this Pull Request:

* Adds Mastodon to the Social Menu and Social Icons Widget

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Adds to an exciting feature

#### Testing instructions:






Go to the Customizer with a theme that supports social menus and add a social menu using the link `mastodon.social`, and do the same with the Social Icons widget, then verify the icon displays as expected.

<img width="301" alt="Screenshot 2020-03-21 at 07 57 48" src="https://user-images.githubusercontent.com/43215253/77222898-81a6c480-6b4f-11ea-967f-c557a1aadc80.png">

#### Proposed changelog entry for your changes:

*Social Menu & Social Icon Widget: Add Mastodon support
